### PR TITLE
Moving simple task file

### PR DESCRIPTION
**Pull request by esthermmoturi**
Moving the [simple tasks](https://docs.communityhealthtoolkit.org/building/tutorials/tasks-1/) file to work on [this](https://github.com/medic/cht-docs/issues/1637).

This is what is expected.
image

### CODE REVIEW

**Comment by andrablaj** [comment]
@esthermmoturi, quick clarification before I review. The plan is to move everything from `Features` and `Tutorials` to `Building`?

**Comment by esthermmoturi** [comment]
> @esthermmoturi, quick clarification before I review. The plan is to move everything from `Features` and `Tutorials` to `Building`?

Yes, that is the plan and to combine scattered concepts. I will also move items from `Concepts`

**Review by andrablaj — CHANGES_REQUESTED** [review]
Images load correctly and links are OK. Just a small comment about wording and a duplicated line.

In general, when building the subtitles of a section, I suggest avoiding the title name in each subtitle. For example, instead of "Building a task", "Configuring a task", etc., we could use "Building", "Configuring" to keep it simple.

File: content/en/building/tasks/simple-tasks.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "Building A Simple Task"
-linkTitle: Tasks
+linkTitle: Simple Tasks
```
[the remark this answers is not in the document]
Comment (esthermmoturi) [reply content/en/building/tasks/simple-tasks.md:3]: I think we can leave this, we have [another file ](https://docs.communityhealthtoolkit.org/building/tutorials/tasks-2/)that I'll be moving on Complex Tasks. What do you think?

[the remark this answers is not in the document]
Comment (andrablaj) [reply content/en/building/tasks/simple-tasks.md:3]: Oooh right, if there is a `Complex Tasks` page, it makes sense to keep it the way you added it. Thank you for the clarification!

**Review by andrablaj — APPROVED** [review]
💯

**Pull request merged**